meta-microservicebus-node/recipes-microservicebus-node/node-pty/node-pty_0.9.0.bb

```diff
@@ -33,7 +33,7 @@ inherit npm
 
```

Comment:
Remove node-pty recipe, msb-core fetch this on demand.

Reply:
Removed the **nodejs-pty** recipe in https://github.com/axians/meta-microservicebus/pull/49/commits/16ef03fa627f06aa6b8dd4353f0702cdbc87acd5